src/Polytopes/Polyhedron/constructors.jl

```diff
@@ -158,7 +172,12 @@ end
 ###############################################################################
 ###############################################################################
 function Base.show(io::IO, P::Polyhedron)
-    print(io, "A polyhedron")
+    ad = ambient_dim(P)
+    if ad == -1.0
```

Comment:
Why is the ambient dimension compared to a float, is there some subtlety I am missing here?
```suggestion
    if ad == -1
```